add docs for dex users copying their CA (#693)
OSC/ood-documentation: source/installation/add-ssl.rst

      For documentation on SSL directives please see:
      https://httpd.apache.org/docs/2.4/mod/mod_ssl.html

2. Update CA (Dex Users only)
`````````````````````````````

Dex users may encounter issues with SSL certificates like::

  remote error: tls: unknown certificate authority

If this is the case, you need to ensure that their certificate authority (CA) is
in the system trust store.

Copy your certificate authority (examples could be ``fullchain.pem``)
to your trust store. replace ``CA CERT location`` with the actual certificate
authority you're using and run these commands to copy it to the appropriate place.

.. tabs::

  .. tab:: RHEL

    .. code:: sh

      sudo cp <CA CERT location> /etc/pki/ca-trust/source/anchors/
      sudo update-ca-trust extract

>       sudo yum update ca-certificates

Reviewer: Don't need to update ca-certificates package to add custom CA to trust store.  There are cases where updates to `ca-certificates` package alone will clear up CA issues if someone uses a cert from newer CA and that CA is included in newer `ca-certificates` package.  If that package update doesn't solve the issue, then a site would need to update the global trust store.

Reviewer: Also there is a Ubuntu trust store package too, blanking on name, but maybe good first step is document that package update for RHEL/Ubuntu, and if that does not solve issue, take steps to add CA to trust store.

Author: Yea just trying to make it fool proof. I'll pull that out into it's own spot.

  .. tab:: Ubuntu

    .. code:: sh

      sudo cp <CA CERT location> /usr/local/share/ca-certificates/
      sudo update-ca-certificates

> 3. Restart the Apache service for the changes take effect.

Reviewer: If this is for Dex, only need to restart `ondemand-dex` service.  Apache has ways to configure custom CAs so not necessary to update global trust store for Apache.

Reviewer: Nevermind, I see this #3 is after the dex things.

      For documentation on SSL directives please see:
      https://httpd.apache.org/docs/2.4/mod/mod_ssl.html

2. Update CA (Dex Users only)
`````````````````````````````

Dex users may encounter issues with SSL certificates like::

  remote error: tls: unknown certificate authority

If this is the case, you need to ensure that their certificate authority (CA) is
in the system trust store and that your CA certificates are up to date.

First, try updating your CA certificates.  This could especially happen when

> you have a `Let's Encrypt`_ and your machine does not know it's a certificate
> authority.
> 

Reviewer: I think this is worded incorrectly.  Here is possible suggestion:
```suggestion
you have a `Let's Encrypt`_ and your machine does not know about that certificate
authority.

```

The certificate authority is LetsEncrypt.

.. tabs::

  .. tab:: RHEL

    .. code:: sh

      sudo yum update ca-certificates

  .. tab:: Ubuntu

    .. code:: sh

      sudo apt update ca-certificates

If you're still having issues, copy your certificate authority (examples could be ``fullchain.pem``)
to your trust store. replace ``CA CERT location`` with the actual certificate
authority you're using and run these commands to copy it to the appropriate place.

.. tabs::

  .. tab:: RHEL

    .. code:: sh

      sudo cp <CA CERT location> /etc/pki/ca-trust/source/anchors/
      sudo update-ca-trust extract

  .. tab:: Ubuntu

    .. code:: sh

>       sudo cp <CA CERT location> /usr/local/share/ca-certificates/

Reviewer: Still need `sudo update-ca-certificates` I believe to update the global trust store with copied CA.  There is a large CA bundle stored in /etc/ that gets updated, same thing happens on RHEL I believe.

Author: thanks

      For documentation on SSL directives please see:
      https://httpd.apache.org/docs/2.4/mod/mod_ssl.html

2. Update CA (Dex Users only)
`````````````````````````````

Dex users may encounter issues with SSL certificates like::

  remote error: tls: unknown certificate authority

If this is the case, you need to ensure that their certificate authority (CA) is
in the system trust store and that your CA certificates are up to date.

First, try updating your CA certificates.  This could especially happen when
you have a `Let's Encrypt`_ and your machine does not know about that certificate
authority.

.. tabs::

  .. tab:: RHEL

    .. code:: sh

      sudo yum update ca-certificates

  .. tab:: Ubuntu

    .. code:: sh

      sudo apt update ca-certificates

If you're still having issues, copy your certificate authority (examples could be ``fullchain.pem``)
to your trust store. replace ``CA CERT location`` with the actual certificate
authority you're using and run these commands to copy it to the appropriate place.

.. tabs::

  .. tab:: RHEL

    .. code:: sh

      sudo cp <CA CERT location> /etc/pki/ca-trust/source/anchors/

>       sudo update-ca-trust extract
>       update-ca-certificates

Reviewer: ```suggestion
      sudo update-ca-trust
```

Reviewer: Finally got around to looking at RedHat docs for this

  .. tab:: Ubuntu

    .. code:: sh

      sudo cp <CA CERT location> /usr/local/share/ca-certificates/

>       update-ca-certificates

Reviewer: ```suggestion
      sudo update-ca-certificates
```
To make things consistent
